docs/docs/references/modes/client-server.md

```diff
@@ -2,6 +2,10 @@
 
 Trivy has client/server mode. Trivy server has vulnerability database and Trivy client doesn't have to download vulnerability database. It is useful if you want to scan images or files at multiple locations and do not want to download the database at every location.
 
+|   Client/Server Mode  | Image | Rootfs | Filesystem | Repository | Config |
+|:---------------------:|:-----:|:------:|:----------:|:----------:|:------:|
+|       Supported       |   ✅  |   X    |     ✅     |     ✅     |  ✅    |
```

Comment:
You can see which target supports client/server.

Rootfs is supported.
```
$ trivy rootfs -h | grep "\-\-server"
      --server string            server address in client mode
```

K8s and AWS are not supported.

```
$ trivy k8s -h | grep "\-\-server"
$ trivy aws -h | grep "\-\-server"
$
```